src/data/faq.json

```
@@ -1521,7 +1543,10 @@
                                     "<b>TAN Hotline</b>",
                                     "The TAN hotline will help you to enter the TAN if you have a positive PCR test result and if you have not received a TAN or document with a PCR-Test QR code. The TAN hotline is available Monday to Sunday from 7 a.m. to 8 p.m. German local time:",
                                     "<b>National:</b> 0800 7540002",
-                                    "<b>International:</b> +49 30 498 75402"
+                                    "<b>International:</b> +49 30 498 75402",
+                                    "Please note that as of version 3.0 of the Corona-Warn-App, teleTAN is no longer required to share a positive test result that has not been transmitted to the Corona-Warn-App. Therefore, the operation of the TAN hotline will be discontinued as of  ?????h.",

```

Specified placeholder was ??????h, one ? is missing
```suggestion
                                    "Please note that as of version 3.0 of the Corona-Warn-App, teleTAN is no longer required to share a positive test result that has not been transmitted to the Corona-Warn-App. Therefore, the operation of the TAN hotline will be discontinued as of ??????h.",
```